MMM-text-clock.js

```diff
@@ -187,25 +190,60 @@ Module.register('MMM-text-clock', {
       grid.style.gridGap = gridGap;
     }
 
-    const words = this.getActiveWords(new Date());
+    const activeWords = this.getActiveWords(new Date());
+    if (this.letters) {
```

Comment:
I think we should do this check based on the language. Arabic is the only language I can think of with this specific ruleset. and then maybe return directly instead of using `else`. So pseudo code:

```javascript
if(this.currentLanguage === 'ar') {
  // ...

  return grid;
}

// .. code for other languages
```

I think it will be a bit more readable and it adds the possibility to add other custom implementations for future languages.

What do you think?

Reply:
There are actually quite a few languages like Hindi, Thai, Armenian, other variants of Arabic... that are similar in nature
I tried to make the implementation generic for future contributors but I can make it specific if you prefer

Reply:
Ah oke, didn't know that, then this is fine!